rotkehlchen/api/v1/schemas.py

```diff
@@ -1756,29 +1758,14 @@ def validate_schema(  # pylint: disable=no-self-use
                 field_name='order_by_attributes',
             )
 
-    @post_load
-    def make_assets_post_query(  # pylint: disable=no-self-use
```

Comment:
As I wrote in the other file, keep this but pass the user db to its initialization so you can properly initialize `AssetsFilterQuery` here